test/uk/gov/hmrc/securemessage/repository/ConversationRepositorySpec.scala

```diff
@@ -106,22 +106,19 @@ class ConversationRepositorySpec extends PlaySpec with MongoSpecSupport with Bef
       result.map(_.conversationId) mustBe List("345", "234", "123")
     }
 
-    "not be returned and instead a database exception raised when a tag filter without an enrolment counterpart is provided" in {
+    "none returned when a tag filter without an enrolment counterpart is provided" in {
       repoSetup()
-      an[reactivemongo.core.errors.DetailedDatabaseException] should be thrownBy {
+      val result =
         await(repository.getConversationsFiltered(Set.empty, Some(List(Tag("notificationType", "CDS Exports")))))
-      }
+      result.size mustBe 0
```

Comment:
you can also use `result mustBe empty`

Reply:
get wartremover complaints [wartremover:Any] Inferred type containing Any: [e]scala.collection.GenTraversable[e]

Reply:
result mustBe Nil works